src/components/TeamPics/teampics.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+
+import TRUMAN from '../../assets/images/truman.jpg';
+import LORENZO from '../../assets/images/lorenzo.jpg';
+import MARY from '../../assets/images/mary.jpg';
+//import SHIV from '../../assets/images/shiv.jpg';
+//import ATHENA from '../../assets/images/athena.jpg';
+//import ADRIAN from '../../assets/images/adrian.jpg';
+//import BUSHER from '../../assets/images/busher.jpg';
+//import TUZETTE from '../../assets/images/tuzette.jpg';
+//import MANJOT from '../../assets/images/manjot.jpg';
+//import KACEY from '../../assets/images/kacey.jpg';
+//import SID from '../../assets/images/sid.jpg';
+//import INTISER from '../../assets/images/intiser.jpg';
+
+import './teampics.scss';
+
+
+/* Outline for this component largely taken from: https://www.codesmite.com/article/how-to-create-pure-css-hexagonal-grids */
+
+
+const Test: React.FC = () => {
+    return (
+        <body>
+            <div className="Header">
+                <h1>Our Team</h1>
+            </div>
+
+            <ul id="grid" className="clear">
```

Comment:
(Optional) Maybe try doing this dynamically instead of hardcoding each image, so when we add or remove team members we don't have to go back and edit the design and structure.

Reply:
For design and structure: The website I used for the honeycomb design had it so the default layout of the honeycomb is 3 hexagons per row. By changing lines 22 and 23 of teampics.scss, we can limit the original sizes of the hexagons, thus changing how many hexagons can be shown per row. Then line 6 scales up or down all the hexagons. 

I don't know what you mean about not hardcoding in each new image.

Reply:
oh I was more thinking of it in the .tsx file in the HTML portion